ci/solana-version.sh

```diff
@@ -14,7 +14,7 @@
 if [[ -n $SOLANA_VERSION ]]; then
   solana_version="$SOLANA_VERSION"
 else
-  solana_version=v1.8.14
+  solana_version=v1.9.5
```

Comment:
Updating the version was necessary as `load_instruction_at_checked` is not available in `1.8.14`. I opted to update to the latest, but we could discuss whether or not that makes sense.

Reply:
i think 1.9 is not on mainnet yet though?

Reply:
yeh i just checked and 1.8.14 already has `load_instruction_at_checked` so maybe lets go with that